proto/proto/penumbra/core/transaction/v1alpha1/transaction.proto

```diff
@@ -81,6 +81,9 @@ message Action {
 message TransactionPerspective {
   repeated PayloadKeyWithCommitment payload_keys = 1;
   repeated NullifierWithNote spend_nullifiers = 2;
+  // The openings of note commitments referred to in the transaction
+  // but not included in the transaction.
+  repeated crypto.v1alpha1.Note advice_notes = 3;
```

Comment:
Adding view support to the new `SwapClaim` where only the commitments, not encrypted notes, are present required adding a new piece of data to the `TransactionPerspective`, to be able to supply the openings of rolled-up state commitments.

Reply:
We can also use this for, e.g., a future `RolledUpOutput` action that doesn't include the encrypted note in the compact block.